### PR TITLE
test babel-preset-yoshi

### LINKED ISSUE #1221: Test babel-preset-yoshi
### Summary

Currently, we have no tests for `babel-preset-yoshi`. If or when things break, we'll know about it only after we release or merge into `master`. We did in fact break recently with legacy decorators.

We can add tests fairly easy to every supported feature by adding use cases (features) to our kitchensink tests (https://github.com/wix/yoshi/tree/master/test/projects) and making sure they render correctly in the browser.

### PR DESCRIPTION
**Pull request by koretskiyav**
### 🔦 Summary
fix #1221

### 🗺️ Test Plan
this is the test

### CODE REVIEW

File: test/projects/kitchensink-javascript/integration/babel-preset-yoshi.test.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      );
+    });
+
+    expect(list['instance-property']).toEqual('instance-property');
+    expect(list['instance-function']).toEqual('instance-function');
+    expect(list['instance-bound-function']).toEqual('instance-bound-function');
+    expect(list['static-property']).toEqual('static-property');
+    expect(list['static-function']).toEqual('static-function');
+    expect(list['static-bound-function']).toEqual('static-bound-function');
+    expect(list['two-in-the-cube']).toEqual('8');
```
Comment (yavorsky) [inline test/projects/kitchensink-javascript/integration/babel-preset-yoshi.test.js]: I would use not the actual result, but the feature we want to test. For example, here we want to check `exponential operator (**)`.

File: test/projects/kitchensink-javascript/src/components/features/assets/babel-preset-yoshi.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+// @babel/preset-env
+export const twoInTheCube = 2 ** 3;
+
+export const asyncDouble = async num => {
+  await (() => {})();
+  return num * 2;
+};
```
Comment (yavorsky) [inline test/projects/kitchensink-javascript/src/components/features/assets/babel-preset-yoshi.js]: Why do we want to test double here, but not just async result? :)

File: test/projects/kitchensink-javascript/src/components/features/assets/babel-preset-yoshi.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+//@babel/plugin-proposal-class-properties
```
Comment (yavorsky) [inline test/projects/kitchensink-javascript/src/components/features/assets/babel-preset-yoshi.js]: Does it mean something ? :) If yes, then why we have no comment for dynamic import?

Comment (koretskiyav) [reply test/projects/kitchensink-javascript/src/components/features/assets/babel-preset-yoshi.js]: added dynamic import

File: test/projects/kitchensink-javascript/src/components/features/assets/babel-preset-yoshi.js (hunk 2 above)
Comment (koretskiyav) [reply test/projects/kitchensink-javascript/src/components/features/assets/babel-preset-yoshi.js]: fixed

File: test/projects/kitchensink-javascript/integration/babel-preset-yoshi.test.js (hunk 1 above)
Comment (koretskiyav) [reply test/projects/kitchensink-javascript/integration/babel-preset-yoshi.test.js]: fixed

**Pull request closed without merging**